src/backend/api/Fusion.Resources.Domain/Queries/GetResourceAllocationRequestItem.cs

```diff
@@ -222,14 +223,25 @@ private async Task ExpandResourceOwnerAsync(QueryResourceAllocationRequest reque
                     if (request.ProposedPerson?.AzureUniqueId is not null)
                     {
                         var manager = await mediator.Send(new GetResourceOwner(request.ProposedPerson.AzureUniqueId));
-                        request.ProposedPerson.ResourceOwner = manager;
+
+                        if (manager != null)
```

Comment:
manager?.FullDepartment is not null instead? 
A manager MAY not have FullDepartment populated